### PR TITLE
Changed input factor of max file size in settings to MB

### PR DESCRIPTION
**Pull request by warpech**
This is a retargeted PR #42 with two fixes of mine.

It solves the issue: https://github.com/StarcounterPrefabs/Images/issues/41

@Mnosek please review.

Merging `SweOffshore-Checklist-UriMapping` to `develop` will be a next step priority that will require joint effort of @diliger and @aloscha. I will send an email about that.

### CODE REVIEW

**Comment by warpech** [comment]
@jwillmer please confirm if you agree with my thinking that 1 MB should be regarded as 1 000 000 bytes?

**Comment by jwillmer** [comment]
I prefer to use the real conversion values. People who don't know it wont mind but people who do get confused if you use something else.

**Comment by jwillmer** [comment]
In regards to your research I don't mind anymore. Writing MB and meaning MiB is O.K. with me.

**Comment by warpech** [comment]
Ok, then I will do what I think is right, that is change the unit to MiB and write MiB.

**Comment by warpech** [comment]
Unit changed to KiB/MiB.

@aloscha please review and merge if all ok.

File: src/Images/Helpers/IllustrationHelper.cs [hunk 1]
```diff
@@ -44,9 +44,19 @@ public string GetUploadDirectoryWithRoot()
             return GetUploadRoot() + GetUploadDirectory();
         }
 
-        public int GetMaximumFileSize()
+        public decimal GetMaximumFileSizeMiB()
```
Comment (aloscha) [inline src/Images/Helpers/IllustrationHelper.cs]: I really confused. Here we have two methods to do Get and Set, why we just create an field for that?
MaximumFileSizeMiB
{
get{...}
set{...}
}

**Comment by aloscha** [comment]
@warpech I find just one thing, please have a look of that

File: src/Images/Helpers/IllustrationHelper.cs (hunk 1 above)
Comment (warpech) [inline src/Images/Helpers/IllustrationHelper.cs]: I just cherry-picked what @jwillmer did... I think the best idea would be to replace that completely with a helper method `BytesToMiB` and `MiBToBytes`.

Comment (aloscha) [inline src/Images/Helpers/IllustrationHelper.cs]: Agreed

**Comment by warpech** [comment]
> @warpech I find just one thing, please have a look of that

Fixed. Please take a look.

**Comment by aloscha** [comment]
For me looks perfect. Please merge :)

**Comment by warpech** [comment]
Merged, thanks for the review

**Pull request merged**